### PR TITLE
fix: move toTextStream to affine

### PR DESCRIPTION
**Pull request by pengx17**

### CODE REVIEW

**Comment by pengx17** [comment]
* **#6628** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/6628?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @pengx17 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/provider.ts [hunk 1]
```diff
@@ -260,7 +255,7 @@ export function setupAIProvider() {
     if (options.attachments?.length) {
       promptName = 'debug:action:fal-sd15';
     }
-    return toImage({
+    return textToText({
```
Comment (fundon) [inline packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/provider.ts]: In the case of  text or image, image to image, need to call `imagesStream`

Comment (pengx17) [reply packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/provider.ts]: both types of message have been listened to in https://github.com/toeverything/AFFiNE/pull/6628/files#diff-d2d8b857ef3f0fefca4175360625704b75151205862d68760a5baf53d196c179R48-R49 and mapped in https://github.com/toeverything/AFFiNE/pull/6628/files#diff-e8a9e3cb567d0cc9ab5e33a5ad6cba230be641ad804f7b8440c852fcdbb98333R126-R128

File: packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts [hunk 2]
```diff
@@ -152,30 +154,3 @@ export function textToText({
...
-      const { messageId, sessionId } = await createSessionMessage({
-        docId,
-        workspaceId,
-        promptName,
-        content,
-        attachments,
-        params,
-      });
-
-      const eventSource = client.imagesStream(messageId, sessionId);
```
Comment (fundon) [inline packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts:177]: both `make it real` and `create an image` need it

Comment (fundon) [reply packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts:177]: If the backend has already merged the `stream` and `images` interfaces into one. ignore it

Comment (pengx17) [reply packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts:177]: I see. Looks like we still need this function. Thanks!

Comment (pengx17) [reply packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts:177]: `make it real` uses `textToText` before this PR though

**Review by fundon — APPROVED** [review]

**Pull request merged**